lms/djangoapps/courseware/models.py

```diff
@@ -68,11 +69,20 @@ def chunked_filter(self, chunk_field, items, **kwargs):
         return res
 
 
+class ManagerWithChunkingCallStackManager(CallStackManager, ChunkingManager):
```

Comment:
Nit: Perhaps just `ChunkingCallStackManager`? Then you don't repeat `Manager` in the name.